vispy/geometry/curves.py

```diff
@@ -0,0 +1,343 @@
+# ----------------------------------------------------------------------------
+#  Anti-Grain Geometry (AGG) - Version 2.5
+#  A high quality rendering engine for C++
+#  Copyright (C) 2002-2006 Maxim Shemanarev
+#  Contact: mcseem@antigrain.com
+#           mcseemagg@yahoo.com
+#           http://antigrain.com
+#
+#  AGG is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
```

Comment:
I'm pretty sure translation of code (C++ -> Python) would still be disallowed by GPL since it would be a derivative work. AFAIK you can use concepts but not the actual code, otherwise you have to obey the license.

But since you translated the 2.4 we should be okay. @sh4wn please remove the GPL notice, then, since it should not actually apply to this code (or the geometry code, I think).